lib/Basic/PlatformUtility.cpp

```diff
@@ -20,7 +20,7 @@
 using namespace llbuild;
 using namespace llbuild::basic;
 
-bool sys::chdir(const char* fileName) {
+bool sys::chdir(const char *fileName) {
```

Comment:
I follow the Google style for placement of `*`, so the previous version was correct.